### DISCUSSION TITLE
Browser Extension

### DISCUSSION THREAD

**Discussion by Bashamega** [EddieHubCommunity/RepoRater#145] [Ideas]
Hello:)

I think many of us just like to browse cool repos on GitHub, and like them, but they are to lazy to go to rate it, all if I want to see a repo rating without going and searching for the repo. I propose building a browser extension that can display the rating. Also, we can let the user login to his GitHub repo account in the browser extension, so he can just rate it for inside of the repo.
Similar to the [Open with GitPod](https://chromewebstore.google.com/detail/gitpod/dodmmooeoklaejobgleioelladacbeki?hl=en&pli=1) browser extension.

**Comment by Bashamega** [comment]
@eddiejaoude

**Reply by eddiejaoude** [reply]
Thank you for the suggestion, but this already exists https://github.com/EddieHubCommunity/RepoRater-Chrome

Maybe we need to add more info about it to this project and a link?

**Reply by Bashamega** [reply]
Yes it will be better if we could add like a `download extenstion` button on the main page

**Reply by eddiejaoude** [reply]
yes that is a good idea 👍  please raise an issue for this
